config/longrun_configs/amip_target_diagedmf.yml

```diff
@@ -12,8 +12,8 @@ aerosol_radiation: true
 prescribed_aerosols: ["CB1", "CB2", "DST01", "DST02", "DST03", "DST04", "DST05", "OC1", "OC2", "SO4", "SSLT01", "SSLT02", "SSLT03", "SSLT04", "SSLT05"]
 surface_setup: "DefaultMoninObukhov"
 turbconv: "diagnostic_edmfx"
+tracer_upwinding: first_order
```

Comment:
This is not used in 0M so it's not changing anything, but it's fine to have it too. (same for all other 0M yamls)